default/python/AI/AIDependencies.py

```diff
@@ -77,21 +77,21 @@
 # boni that scale with planet size
 POP_SIZE_MOD_MAP = {
     # "Effect": [uninhab, hostile, poor, adequate, good]
-    "environment_bonus": [0,-4,-2, 0, 3],
-    "GRO_SUBTER_HAB":    [0, 1, 1, 1, 1],
+    "environment_bonus": [0, -4, -2, 0, 3],
+    "GRO_SUBTER_HAB": [0, 1, 1, 1, 1],
     "GRO_SYMBIOTIC_BIO": [0, 0, 1, 1, 1],
     "GRO_XENO_GENETICS": [0, 1, 2, 2, 0],
-    "GRO_XENO_HYBRID":   [0, 2, 1, 0, 0],
-    "GRO_CYBORG":        [0, 2, 0, 0, 0],
-    "CON_NDIM_STRUC":    [0, 2, 2, 2, 2],
-    "CON_ORBITAL_HAB":   [0, 1, 1, 1, 1],
+    "GRO_XENO_HYBRID": [0, 2, 1, 0, 0],
+    "GRO_CYBORG": [0, 2, 0, 0, 0],
+    "CON_NDIM_STRUC": [0, 2, 2, 2, 2],
+    "CON_ORBITAL_HAB": [0, 1, 1, 1, 1],
```

Comment:
Ah, now I just got down to seeing that this PR also has changes to universe_tables.py  -- the tables there are exactly the kind of situation where it is very important to be able to compare within columns across rows, and so I think we should either exclude it entirely from the PEP8 review or at least bypass the whitespace/alignment rules for it.